fuzzer/src/main/scala/firrtl/ExprGen.scala

```diff
@@ -681,6 +681,28 @@ object ExprGen {
     }
   }
 
+  object MuxGen extends ExprGen[Mux] {
+    def name = "mux"
+    private def imp[S: ExprState, G[_]: GenMonad](tpe: Type): StateGen[S, G, Mux] = {
+      for {
+        cond <- ExprState[S].exprGen(Utils.BoolType)
+        expr1 <- ExprState[S].exprGen(tpe)
+        expr2 <- ExprState[S].exprGen(tpe)
```

Comment:
The spec seems to say that "The types of the two input expressions are equivalent". However in practice it seems like we happily except something like this:
```.fir
circuit m:
  module m:
    input a : UInt<1>
    input b : UInt<2>
    input sel: UInt<1>
    output c: UInt
    c <= mux(sel, a, b)
```

Thus it would probably make sense to relax the type requirements here.